vueapp/components/Videos/VideosTable.vue

```diff
@@ -171,6 +169,7 @@
 import { mapGetters } from "vuex";
 import StudipButton from "@studip/StudipButton";
 import StudipIcon from "@studip/StudipIcon";
+import StudipProgressIndicator from "@studip/StudipProgressIndicator.vue";
 
 import VideoRow from './VideoRow.vue';
 import EmptyVideoRow from './EmptyVideoRow.vue';
```

Comment:
This should also be removed!
As well as in the components listing: https://github.com/elan-ev/studip-opencast-plugin/blob/5f130981a428f11356bb8fc30593b7998b6b62ad/vueapp/components/Videos/VideosTable.vue#L204C35-L204C48